src/Adapter/Country/CountryDataProvider.php

```diff
@@ -64,7 +64,7 @@ public function getCountriesIdWhichNeedDni()
             ->from('country', 'c')
             ->where('c.`need_identification_number` = 1')
         ;
-        $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS($query);
+        $result = Db::getInstance((bool) _PS_USE_SQL_SLAVE_)->executeS($query);
```

Comment:
As i said earlier, change the const value instead

Reply:
Fixed